src/object/C4Movement.cpp

```diff
@@ -186,50 +227,74 @@ void C4Object::VerticalBounds(C4Real &ctcoy)
 		{
 			C4Real tbound = itofix(Layer->GetY() + Layer->Shape.GetY() - Shape.GetY()),
 			       bbound = itofix(Layer->GetY() + Layer->Shape.GetY() + Layer->Shape.Hgt - (Shape.GetY() + Shape.Hgt));
-			if (ctcoy < tbound) StopAndContact(ctcoy, tbound, ydir, CNAT_Top);
-			if (ctcoy > bbound) StopAndContact(ctcoy, bbound, ydir, CNAT_Bottom);
+			if (ctcoy < tbound)
+			{
+				StopAndContact(ctcoy, tbound, ydir, CNAT_Top);
+			}
+			if (ctcoy > bbound)
+			{
+				StopAndContact(ctcoy, bbound, ydir, CNAT_Bottom);
+			}
 		}
 	}
 	// landscape bounds
 	C4Real tbound = itofix(0 - Shape.GetY()),
 	       bbound = itofix(::Landscape.GetHeight() - (Shape.GetY() + Shape.Hgt));
 	if (ctcoy < tbound && GetPropertyInt(P_BorderBound) & C4D_Border_Top)
+	{
 		StopAndContact(ctcoy, tbound, ydir, CNAT_Top);
+	}
 	if (ctcoy > bbound && GetPropertyInt(P_BorderBound) & C4D_Border_Bottom)
+	{
 		StopAndContact(ctcoy, bbound, ydir, CNAT_Bottom);
+	}
 }
 
 void C4Object::DoMovement()
 {
-	int32_t iContact=0;
-	bool fAnyContact=false; int iContacts = 0;
-	BYTE fTurned=0,fRedirectYR=0,fNoAttach=0;
+	int32_t iContact = 0;
+	bool fAnyContact = false;
+	int iContacts = 0;
+	BYTE fTurned = 0,
+		 fRedirectYR = 0,
+		 fNoAttach = 0;
```

Comment:
Can you turn these into individual declarations, per http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-name-one?

Reply:
Yes, I prefer that, but was not sure what is better. Good to know!